src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue

```diff
@@ -9,27 +9,51 @@
             <dashboard-toolset />
         </div>
         <p-divider />
-        <dashboard-refresh-dropdown :interval-option.sync="state.refreshInterval"
-                                    refresh-disabled
-        />
+        <div>
+            <p-button icon-left="ic_plus"
+                      style-type="highlight"
+                      @click="handlerOpenOverlay"
```

Comment:
suggestion : According to our custom, it would be better to change from 'handler' to 'handle'!

Reply:
Oh... mistake...  thanks!